app/app.iml

```diff
@@ -4,6 +4,8 @@
     <facet type="android-gradle" name="Android-Gradle">
       <configuration>
         <option name="GRADLE_PROJECT_PATH" value=":app" />
+        <option name="LAST_SUCCESSFUL_SYNC_AGP_VERSION" />
```

Comment:
Don't commit changes to the AndroidStudio project files (as they are modified every time its run). They are only included in the repo to assist opening the project for the first time (and should arguably should be removed as they are completely optional to the build process and just making a mess).